Catalog/TableDescriptor.h

```diff
@@ -54,6 +54,9 @@ struct TableDescriptor {
   int64_t maxChunkSize;    // max number of rows per fragment
   int32_t fragPageSize;    // page size
   int64_t maxRows;         // max number of rows in the table
+  int64_t skipRows;        // number of skipped rows of data in CSV file
```

Comment:
I'd like to find a different place for this -- one option might be to create a `DataframeTableDescriptor` which could inherit from TD but supply the addition options for transport to FSI. The TableDescriptor API is already too big -- it should ideally contain just the base id, shard, etc for the table.